gallery_dl/extractor/kemonoparty.py

```diff
@@ -69,9 +69,8 @@ def items(self):
 
             for post["num"], file in enumerate(files, 1):
                 post["type"] = file["type"]
-                url = file["path"]
                 if url[0] == "/":
-                    url = self.root + url
+                    url = self.root + '/data' + url.replace(self.root, '')
```

Comment:
URLs starting with a `/` don't also contain or start with `https://kemono.party`.

I'd rather add another check and handle this there
```suggestion
                    url = self.root + "/data" + url
                elif url.startswith("https://kemono.party"):
                    url = self.root + "/data" + url[20:]
```